WordPress/twentytwentyone
# Fix menu icon fill colors on dark backgrounds

Fixes #360 

## Summary

Since the icons are on the `background` of buttons, the only way to tweak them was to set a different icon when the background-color is dark (can't use CSS `'fill`).
The only way that could happen is if we have a way to differentiate dark from light, so a body-class `is-dark-background` gets added when needed.
This PR adds the class in `<body>` both in PHP and in the customizer when changing the background-color.
After that was done, the CSS was tweaked to accommodate the white icons on dark backgrounds.

## Test instructions

This PR can be tested by following these steps:
1. Go to the customizer > colors, and select a dark background. You should see the menu icons change color to white.
2. Save, exit the customizer and verify on the frontend.
3. Go back to the customizer and select a light background. You should see the menu icons change color to dark.
4. Save, exit the customizer and verify on the frontend.

## Quality assurance
